errutil/error.go

```diff
@@ -4,6 +4,11 @@
 // Utilities include:
 //
 // - An error type that makes it easy to work with const error sentinels.
+// - An easy way to wrap a list of errors together.
+// - An easy way to merge a list of errors opaquely.
+//
+// Flexible enough that you can do your own wrapping/merging logic
```

Comment:
For this flexibility, it lacks a `MergeReduce(reducer, errs...)` function, no?

Reply:
I don't think so, you have errutil.Reduce() where you can do whatever you want with the errors, no ? (like merging/wrapping or something else).

Reply:
ah yes.. sorry haha